src/main/java/uk/gov/pay/api/ledger/resource/TransactionsResource.java

```diff
@@ -30,9 +31,7 @@ public TransactionsResource(TransactionSearchService transactionSearchService) {
     @Timed
     @Path("/v1/transactions")
     @Produces(APPLICATION_JSON)
-    public TransactionSearchResults getTransactions(@Auth Account account,
-                                                    @BeanParam TransactionSearchParams searchParams) {
-        
+    public SearchResults<PaymentForSearchResult> getTransactions(@Auth Account account, @BeanParam TransactionSearchParams searchParams) {
```

Comment:
Curiously this endpoint seems to return payments or refunds but squashes the transaction into a payment shape. This seems to largely work but doesn't make any sense. 

Making this a generic interface just points this out, the behaviour should continue